files/build_templates/init_cfg.json.j2

```diff
@@ -26,7 +26,9 @@
                    ("snmp", "enabled", true, "enabled"),
                    ("swss", "enabled", false, "enabled"),
                    ("syncd", "enabled", false, "enabled"),
+{% if sonic_asic_platform == "vs" %}
```

Comment:
Just checking, should we have it as "include_gbsyncd" and add as a compilation option?

Reply:
in the future yes, not right now. gbsyncd currently is hardcoded to vs platform.

Reply:
btw, in the future, the gbsyncd should be by default embedded into the the image and loaded at the runtime based on the platform.

Reply:
ok